src/Main.java

```diff
@@ -0,0 +1,400 @@
+import java.util.Scanner;
+import java.util.Random;
+public class Main {
+    public static void main(String[] args) throws InterruptedException {
+        Scanner playerInput = new Scanner(System.in);
+        Random generator = new Random();
+        Character player = new Character();
+        Character zombie = new Character();
+        Character ghost = new Character();
+        Character goblin = new Character();
+        Character dungeonBoss = new Character();
+
+        String[] monsters = {"Zombie", "Ghost", "Goblin"};
+        int rMonster = generator.nextInt(monsters.length);
+        String randomMonster = monsters[rMonster];
+
+        String[] monstersRepeatable = {"Zombie", "Ghost", "Goblin"};
+        int rMonsterRepeatable = generator.nextInt(monstersRepeatable.length);
+        String randomMonsterRepeatable = monstersRepeatable[rMonsterRepeatable];
+
+
+        int monsterAttack = generator.nextInt(6);
+        if (monsterAttack == 0) {
+            monsterAttack = 1;
+        }
+        int monsterHealth = generator.nextInt(16);
+        if (monsterHealth == 0) {
+            monsterHealth = 1;
+        }
+
+        int playerRecover1 = generator.nextInt(11);
+        if (playerRecover1 == 0) {
+            playerRecover1 = 1;
+        }
+        int playerRecover2 = generator.nextInt(6);
+        if (playerRecover2 == 0) {
+            playerRecover2 = 1;
+        }
+        int playerStrength = generator.nextInt(6);
+        if (playerStrength == 0) {
+            playerStrength = 1;
+        }
+
+        //Health, attack, stamina(every monster that gets attacked drains one stamina, if you run away you lose an extra stamina point)
+
+        //Player Stats
+        player.attack = 5;
+        player.health = 20;
+        player.stamina = 20;
+        int h = 1;
+        int i = 1;
+
+        //Zombie Stats
+        zombie.name = "Zombie";
+        zombie.attack = 2;
+        zombie.health = 10;
+
+        //Ghost Stats
+        ghost.name = "Ghost";
+        ghost.attack = 2;
+        ghost.health = 10;
+
+        //Goblin Stats
+        goblin.name = "Goblin";
+        goblin.attack = 2;
+        goblin.health = 10;
+
```

Comment:
consider doing this with parameters in the functions initializer. https://www.geeksforgeeks.org/the-initializer-block-in-java/